### ISSUE TITLE
why my result radar picture totally black?

### ISSUE THREAD

**Issue by bloodywolfb** [cxxixi/Precipitation-Nowcasting#3]
Hi, I run the training.py and use the test.py to predict some test picture in testset. Then I got a totally black picture.So I try to use the model in logs/model to run test.py and can generate a radar picture has radar echo. I try a lot but can't reappear your result. Looking forward for your reply. Thanks a lot.

**Comment by cxxixi** [comment]
Have you checked your results by looking at the pictures you fed in for prediction cause some of the test pictures  do  have little radar echo which might result in totally 'black' sequential pictures in prediction.

**Comment by cxxixi** [comment]
Have you checked your results by looking at the pictures you fed in for prediction cause some of the test pictures do have little radar echo which might result in totally 'black' sequential pictures in prediction.

**Comment by simengy** [comment]
Same here. All hyper-parameters are kept as default, and the model only produced black images. However the model provided can produce the forecasts normally.

**Comment by cxxixi** [comment]
> Same here. All hyper-parameters are kept as default, and the model only produced black images. However the model provided can produce the forecasts normally.

That sounds weird. How is your training log like? It's a good idea to print out some of the training process to check if the model is actually learning or not. Visualization of the gates in LSTM or the images of each time step can be good indicators for that. Besides, based on my memory, the model took quite a while to reach the point where it was able to gave a decent prediction and actually the model 42 I provided in the logs took a couple of hours to train. Since I don't have further information, I guess the training time might be a problem.

**Comment by simengy** [comment]
Thanks for the reply. I basically used the default parameters for the re-training. The training loss did not going down. tried to switch the learning rate "--lr" among [1e-4, 1e-3, 1e-2], the loss did go down. part of the loss records with the default parameters provided as follows:

Epoch: 0 ,Iteration: 100, Duration 43 s, Loss: 58.174644
Epoch: 0 ,Iteration: 200, Duration 34 s, Loss: 59.314331
...
Epoch: 1 ,Iteration: 1600, Duration 37 s, Loss: 62.065887
Epoch: 1 ,Iteration: 1700, Duration 37 s, Loss: 53.423523
...
Epoch: 23 ,Iteration: 100, Duration 37 s, Loss: 69.560722
Epoch: 23 ,Iteration: 200, Duration 37 s, Loss: 62.305500
...
Epoch: 23 ,Iteration: 1600, Duration 36 s, Loss: 59.953064
Epoch: 23 ,Iteration: 1700, Duration 36 s, Loss: 66.148499
Epoch: 24 ,Iteration: 100, Duration 38 s, Loss: 60.604664
Epoch: 24 ,Iteration: 200, Duration 36 s, Loss: 60.631847
...
Epoch: 24 ,Iteration: 1600, Duration 37 s, Loss: 65.840111
Epoch: 24 ,Iteration: 1700, Duration 37 s, Loss: 53.431824

**Comment by GarryLai** [comment]
I meet the same problems, I hope that you can help me.
Thank you very much.

**Comment by astEdd** [comment]
i want to mail you
How can I contact with you?
By fb?
https://www.facebook.com/shichao.lin.14
